WordPress/MediaEditor/Capabilities/Crop/MediaEditorCrop.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import TOCropViewController
+
+class MediaEditorCrop: MediaEditorCapability {
```

Comment:
Since this capability includes crop / rotate / zoom maybe a better name would be MediaEditorBasicCapability or MediaEditorBasic

Reply:
Since we're going to release that as an open-source library I was wondering: does a regular user will understand what Basic means?

What do you think about `MediaEditorCropZoomRotate`? It's big — I know — but represents clearly what it is.

Reply:
MediaEditorCropZoomRotate - I like it

Reply:
Great! As we talked, I'll change this name in the next PR.